stream-chat-android/src/main/java/com/getstream/sdk/chat/viewmodel/channels/ChannelsViewModel.kt

```diff
@@ -92,7 +93,9 @@ public class ChannelsViewModel(
     }
 
     public fun leaveChannel(channel: Channel) {
```

Comment:
Same here, if we could touch upon kdocs whenever we make changes that would make it easier on us to have consistent code documentation in the whole project.